cognitedata/reveal
# feat(react-components): Add support for CDM based 360 images in Image360CollectonContainer

#### Type of change
![Feat](https://img.shields.io/badge/Type-Feat-green)  

## Description :pencil:
Adds support for CDM based 360 images in `Image360CollectonContainer` and bumps Reveal version to `4.22.0`

## How has this been tested? :mag:
Have tested the `Image360CollectonContainer` storybook with `events`, `dm` and `cdm` based 360 images.

## Test instructions :information_source:
Verify that loading cdm based 360 images in the `Image360CollectonContainer` storybook works. Use the following options:
```
    addImage360CollectionOptions: {
      source: 'cdm',
      externalId: 'c_RC_2',
      space: 'anders_360_image',
      transform: new Matrix4()
    }
```


## Checklist :ballot_box_with_check:
